### ISSUE TITLE
AED symbol associated with All-Terrain Vehicle Trail in Editor

### ISSUE THREAD

**Issue by jakecoolidge** [nationalparkservice/places-editor#102]

**Comment by cwlawlis802** [comment]
Note: associated with the point, not line.

Currently set to `npmap-symbol-library` > `All-terrain trail`, which does not exist (so Editor defaults to the `aed` symbol).

I can't seem to find a fitting maki icon, so we may have to default to a generic POI marker unless you can find something else @jakecoolidge. cc @jimmyrocks

**Comment by cwlawlis802** [comment]
I have removed these values for the `iconSource` and `makiIcon` fields in [`NPS_Preset_Classes`](https://docs.google.com/spreadsheets/d/1XFqkmIYMEp73q9flgNto9QJim6N5hbacgkUAgdf0rhE/edit#gid=0).

**Comment by jakecoolidge** [comment]
@cwlawlis802: HFC has this symbol in their extended, in-house set:

![hfc-atv-symbol](https://cloud.githubusercontent.com/assets/11392872/11845564/c2e3e544-a3d1-11e5-9732-4ef47b64d8cd.png)

If we anticipate needing this symbol, we can get it into the symbol library queue. Let me know what you guys think and I'll get an issue going for it. We'll get it into release 2.2.

**Comment by cwlawlis802** [comment]
That looks good to me - I think it would be a nice addition to the NPMap Symbol Library for sure.

**Comment by jakecoolidge** [comment]
Agreed. Issue is [now live](https://github.com/nationalparkservice/npmap-symbol-library/issues/54).

**Comment by cwlawlis802** [comment]
The symbol has been created and the NPS_Preset_Classes sheet has been updated using `all-terrain-trail`.

@jimmyrocks will regenerate presets, which should resolve this.
